src/lib/pages/backoffice/Patron/PatronDetails/ItemsSearch/ItemsResultsList/ItemsResultsList.js

```diff
@@ -52,56 +52,74 @@ export default class ItemsResultsList extends Component {
       circulationStatus
     ) && invenioConfig.ITEMS.canCirculateStatuses.includes(row.metadata.status)
       ? buttonCheckout
-      : this.renderForceCheckoutModal(row);
+      : this.renderForceCheckoutModal(
+          row,
+          circulationStatus === 'ITEM_ON_LOAN'
+        );
   };
 
-  renderForceCheckoutModal = item => {
+  renderForceCheckoutModal = (item, isItemOnLoan) => {
     const { isModalOpen } = this.state;
     const { patronPid } = this.props;
     const itemPid = {
       type: recordToPidType(item),
       value: item.metadata.pid,
     };
+
     return (
-      <Modal
-        trigger={
-          <Button color="yellow" onClick={this.toggleModal}>
-            Force Checkout
-          </Button>
-        }
-        open={isModalOpen}
-        onClose={this.toggleModal}
-        closeIcon
-      >
-        <Header
-          icon="exclamation triangle"
-          content="Force Checkout of Missing Item"
-        />
-        <Modal.Content>
-          <p>
-            You are creating a loan for a missing item, would you like to
-            continue?
-          </p>
-        </Modal.Content>
-        <Modal.Actions>
-          <Button color="red" onClick={this.toggleModal}>
-            <Icon name="remove" /> No
-          </Button>
-          <Button
-            color="green"
-            onClick={() =>
-              this.onClickCheckoutHandler(
-                item.metadata.document_pid,
-                itemPid,
-                patronPid,
-                true
-              )
+      <>
+        <Modal
+          trigger={
+            <Button
+              color="yellow"
+              onClick={this.toggleModal}
+              disabled={isItemOnLoan}
+            >
+              Force Checkout
+            </Button>
+          }
+          open={isModalOpen}
+          onClose={this.toggleModal}
+          closeIcon
+        >
+          <Header
+            icon="exclamation triangle"
+            content="Force Checkout of Missing Item"
+          />
+          <Modal.Content>
+            <p>
+              You are creating a loan for a missing item, would you like to
```

Comment:
Agree about the missing, there might be other reasons. Something like:
`You are creating a loan for a physical copy that cannot circulate. Are you sure?`